test/topology/test_topology_remove_decom.py

```diff
@@ -135,3 +135,18 @@ async def test_rebuild_node(manager: ManagerClient, random_tables: RandomTables)
     servers = await manager.running_servers()
     await manager.rebuild_node(servers[0].server_id)
     await check_token_ring_and_group0_consistency(manager)
+
+@pytest.mark.asyncio
+async def test_concurrent_removenode(manager: ManagerClient):
```

Comment:
This test is inside the test/topology/ suite, which means that it already starts with 3 servers. Then you're adding 3 additional servers -- ending with 6 in total. It's a bit of a waste of resources for such a small test.

Please either move the test to `test/topology_custom/` suite (which starts with 0 servers) or keep it there but don't add additional servers (instead, operate with the servers you are already provided)

Reply:
Done, using the existing servers now!